ZPastel.API/Controllers/OrderController.cs

```diff
@@ -10,18 +15,49 @@ namespace ZPastel.API.Controllers
     public class OrderController : ControllerBase
     {
         private readonly ILogger<OrderController> _logger;
+        private readonly IOrderService orderService;
+        private readonly OrderConverter orderConverter;
 
-        public OrderController(ILogger<OrderController> logger)
+        public OrderController(
+            ILogger<OrderController> logger, 
+            IOrderService orderService,
+            OrderConverter orderConverter)
         {
             _logger = logger;
+            this.orderService = orderService;
+            this.orderConverter = orderConverter;
         }
 
-        [HttpPost("create",Name = nameof(CreateOrder))]
+        [HttpPost("create", Name = nameof(CreateOrder))]
         [Produces("application/json")]
         [ProducesResponseType(StatusCodes.Status204NoContent)]
-        public async Task<ActionResult> CreateOrder()
+        public async Task<ActionResult> CreateOrder(OrderResource orderResource)
         {
+            var order = orderConverter.ConvertToModel(orderResource);
+
+            await orderService.CreateOrder(order);
+
             return NoContent();
```

Comment:
Create a task to return the Created Order

Reply:
Created Task to return the created order